src/UI/Core/Label.cpp

```diff
@@ -0,0 +1,41 @@
+#include "Label.h"
+
+#include "../../Constants.h"
+#include "../../FontManager.h"
+
+#include "NAS2D/Renderer/Renderer.h"
+#include "NAS2D/Resources/Font.h"
+
+#include "NAS2D/MathUtils.h"
+#include "NAS2D/Utility.h"
+
+#include <locale>
```

Comment:
I don't see any obvious use of `<locale>`.

Reply:
This is probably from the TextField and/or TextArea controls and can be safely removed unless there are plans to make use of it in the future.